recipes/rmlui/3.3/conanfile.py

```diff
@@ -0,0 +1,121 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class RmluiConan(ConanFile):
+    name = "rmlui"
+    description = "RmlUi - The HTML/CSS User Interface Library Evolved"
+    homepage = "https://github.com/mikke89/RmlUi"
+    url = "https://github.com/conan-io/conan-center-index"
+    license = "MIT"
+    topics = ("conan", "css", "gui", "html", "lua", "rmlui")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "build_lua_bindings": [True, False],
+        "disable_rtti_and_exceptions": [True, False],
```

Comment:
Please avoid double negations: `disable_rtti_and_exceptions=False` ==> build with rtti.
So name it `enable_rtti_and_exceptions` or something similar.